notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
<p>could you please add link on dataset here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Added. Thank you